jsrc/adverbs/a.c

```diff
@@ -17,7 +17,7 @@ static A swap2(J jt,A a,A w,A self){DECLF; F2PREFIP; jtinplace = (J)(intptr_t)((
 
 // w~, which is either reflexive/passive or evoke
  A jtswap(J jt, A w){A y;C*s;I n;
- ARGCHK1(w); 
+ if(!w) return 0;
```

Comment:
Ah, sorry. The original issue is unclear. For the `ARGCHK` macros, we actually want to just delete them, not replace them with anything. Is that possible?

Reply:
That was what I gathered from the issue, however it seems at least some functionality relies on the check.

```
98% tests passed, 7 tests failed out of 396

Total Test time (real) = 296.12 sec

The following tests FAILED:
        132 - g232 (SEGFAULT)
        179 - g420stch (SEGFAULT)  -- previously failing on osx
        237 - g631 (SEGFAULT)
        268 - gf (SEGFAULT)
        315 - gq (SEGFAULT)
        387 - gu (SEGFAULT)
        388 - gu0 (SEGFAULT)
Errors while running CTest
```

I'll check, if I can remove some and only leave those that are needed for the existing tests.

The commented out tests also add one more unknown to this.

Reply:
Yea, I would say let's remove as much as possible.